buildlib/pr/io_demo/az-stage-io-demo.yaml

```diff
@@ -48,7 +48,7 @@ steps:
     pgrep -u "$USER" -f 'network-corrupter'
     corrupter_pid=$(pgrep -u "$USER" -f 'network-corrupter')
     echo "corrupter_pid=$corrupter_pid"
-    azure_set_variable "corrupter_pid" "$corrupter_pid"
+    azure_set_variable "corrupter_pid" "${corrupter_pid}"
```

Comment:
Why does it critical to have these brackets here?

Reply:
With Azure is whatever works by trial and error. Azure documentation is like a fantasy book sometimes - detached from the reality.